maps/nerva/obj/nerva_items.dm

```diff
@@ -16,55 +16,125 @@
 	item_state = "headset"
 	ks1type = /obj/item/device/encryptionkey/nerva/nt
 
+/obj/item/device/radio/headset/nervananotrasen/alt
+	name = "nanotrasen bowman headset"
+	desc = "A larger headset for corporate drones."
+	icon_state = "nt_headset_alt"
+	item_state = "nt_headset_alt"
+	ks1type = /obj/item/device/encryptionkey/nerva/nt
+
 /obj/item/device/radio/headset/heads/secondofficer
 	name = "second officer's headset"
 	desc = "The headset of the ICS Nerva's second officer."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks1type = /obj/item/device/encryptionkey/nerva/so
 
+/obj/item/device/radio/headset/heads/secondofficer/alt
+	name = "second officer's bowman headset"
+	desc = "The more comfortable headset of the ICS Nerva's second officer."
+	icon_state = "com_headset_alt"
+	item_state = "com_headset_alt"
+	ks1type = /obj/item/device/encryptionkey/nerva/so
+
 /obj/item/device/radio/headset/heads/firstofficer
 	name = "first officer's headset"
 	desc = "The headset of the ICS Nerva's first officer."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks1type = /obj/item/device/encryptionkey/nerva/cap
 
+/obj/item/device/radio/headset/heads/firstofficer/alt
+	name = "first officer's bowman headset"
+	desc = "The thicker headset of the ICS Nerva's first officer."
+	icon_state = "com_headset_alt"
+	item_state = "com_headset_alt"
+	ks1type = /obj/item/device/encryptionkey/nerva/cap
+
 /obj/item/device/radio/headset/heads/nerva_cap
 	name = "captain's headset"
 	desc = "The headset of the ICS Nerva's captain."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks1type = /obj/item/device/encryptionkey/nerva/cap
 
+/obj/item/device/radio/headset/heads/nerva_cap/alt
```

Comment:
you haven't added this to the nerva captain's locker, which is in nerva_closets.dm at the top of the file.

Reply:
whoops